web/utils/date.js

```diff
@@ -16,4 +17,26 @@ const formatToMonthYear = (val) => {
   ];
   return months[parseInt(month, 10) - 1] + ' ' + year;
 };
-export default formatToMonthYear;
+
+export const monthYearToYYYYMM = (val) => {
+  if (!val) return '';
```

Comment:
If `val` is defined but not a string (or is an empty string), calling `split()` will throw an error.

Consider adding some type checking: `if (!val || typeof val !== 'string') return '';` to be extra sure